networks/README.md

```diff
@@ -0,0 +1,51 @@
+# Networks
+
+## Overview
+
+With this example we demonstrate how to create a single network as well as two networks with one stack.
+
+The separation of network and compute resources allows to create multiple stacks containing instances and 
+volumes that are all connected to the same network. This can be useful if one uses the same network(s) 
+for multiple groups of instances(servers) that need to run separately from other instance groups. 
+Of course the same rule applies for single instances too.
+
+## single network
+
+- one network and subnet
+- one router
+- router <-> subnet bridge
+- uplink to internet (ext-net)
+
+### How to start this setup
+
+You can start the stack using the usual command line:
+
+```
+openstack stack create -t 1.single-network.yaml <stackName>
```

Comment:
Does the `1.` and `2.` prefix have a meaning, e.g. do you first need to run single-network and then two-networks ?

Reply:
hm i find that confusing... why are they in the same folder if they are not related?

Reply:
because it's the example setup library. They are related to same group. Matthias and me thought about a better structure for templates. this is part of it. Other templates will be restructures in future. It's currently a bit messy. :)

Reply:
Think about separating restructuring from adding new stuff. Because multiple different structures is worse than a bad structure IMO.

I will still approve this PR